src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+import ExampleComponent from './example/ExampleComponent?dev-site-example';
+
+# Theming Guide
+
+Welcome to the Terra theming strategy guide. This guide will walk teams through the recommended approach for theming custom components.
+
+## Summary
+
+Terra supports a default, clinical-lowlight-theme, and orion-fusion-theme. This guide is for theming custom components. Custom components are any non Terra-UI component created by teams that need to be integrated into an application using a supported Terra theme. Teams should not create new themes or theme Terra components.
```

Comment:
** pretending o be a consumer** why can't I make my own theme or theme terra? 

it might be worth mentioning we don't consider changes, adding or removing theme vars to be non-passive.